cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -222,7 +222,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<functions-reverse-list, reverse()>>           | List              | Returns a list in which the order of all elements in the original list have been reversed.
 |<<functions-reverse, reverse()>>                | String            | Returns a string in which the order of all characters in the original string have been reversed.
 |<<functions-right, right()>>                    | String            | Returns a string containing the specified number of rightmost characters of the original string.
-|<<functions-round, round()>>                   | Numeric           | Returns the value of a number rounded to the nearest integer.
+|<<functions-round1, round()>>                   | Numeric           | Returns the value of the given number rounded to the nearest integer, with half-way values always rounded up.
```

Comment:
This doesn't seem related to encrypted passwords. Has something unrelated got mixed up in this PR?